### PR TITLE
Fix Minor Content Changes from Source Doc

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

File: content/en/blog/_posts/2019-12-09-csi-volume-migration-beta.md [hunk 1]
```diff
@@ -29,11 +29,12 @@ The Kubernetes team has worked hard to ensure the stability of storage APIs and
...
 
-You must also install the requisite CSI driver on your cluster - instructions for this can generally be found from you provider of choice. CSI migration is available for GCE Persistent Disk and AWS Elastic Block Store in beta as well as for Azure Ffile/Ddisk and Openstack/Cinder in alpha. Kubernetes distributors should look at automating the deployment and management (upgrade, downgrade, etc.) of the CSI Drivers they will depend on.
+You must also install the requisite CSI driver on your cluster - instructions for this can generally be found from you provider of choice. CSI migration is available for GCE Persistent Disk and AWS Elastic Block Store in beta as well as for Azure File/Disk and Openstack Cinder in alpha. Kubernetes distributors should look at automating the deployment and management (upgrade, downgrade, etc.) of the CSI Drivers they will depend on.
+
+To verify the feature flag is enabled and driver installed on a particular node you can get the CSI Node object. You should see the in-tree plugin name of the migrated plugin as well as your [installed] driver in the drivers list.
```
Comment (sftim) [inline content/en/blog/_posts/2019-12-09-csi-volume-migration-beta.md]:
```suggestion
To verify the feature flag is enabled and driver installed on a particular node you can get the CSINode object. You should see the in-tree plugin name of the migrated plugin as well as your [installed] driver in the drivers list.
```

In the API, the object is CSINode (once you've added it to your cluster etc).

File: content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md [hunk 2]
```diff
@@ -238,19 +238,19 @@ spec:
...
+The CSI external-snapshotter sidecar then passes the VolumeSnapshotClass parameters, the source volume ID, and any referenced secret(s) to the CSI driver (in this case `testdriver.csi.k8s.io`) via a CSI `CreateSnapshot` call. In response, the CSI driver creates a new snapshot for the specified volume, and returns the ID for that snapshot. The CSI external-snapshotter sidecar then updates the `snapshotHandle`, `creationTime`, `restoreSize`, and `readyToUse` in the status field of the VolumeSnapshotContent object that represents the new snapshot. For a storage system that needs to upload the snapshot after it has been cut, the CSI external-snapshotter sidecar will keep calling the CSI `CreateSnapshot` to check the status until upload is complete and set `readyToUse` is true.
```
Comment (xing-yang) [inline content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md]: s/set `readyToUse` is true/set `readyToUse` to true

**Comment by xing-yang** [comment]
Changes to snapshot blog looks good.
/lgtm

File: content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ type VolumeSnapshotContentStatus struct {
 	ReadyToUse *bool
 	RestoreSize *int64
 	Error *VolumeSnapshotError
-           SnapshotHandle *string
+  SnapshotHandle *string
```
Comment (yuxiangqian) [inline content/en/blog/_posts/2019-12-09-volume-snapshot-beta.md]: sorry to be picky, the indention seems to be still different, maybe tab v.s. space?

**Comment by onlydole** [comment]
@rawkode - can I get your help in addressing that one nitpick on line 155? I can approve it as soon as that's all set!

**Comment by rawkode** [comment]
@onlydole Fixed, even if it took 3 commits!

If you can squash and merge, lets do that; if you want me to squash locally and push - just say.

Not a problem. I'm jumping in a meeting, but can do it this evening

**Comment by onlydole** [comment]
Thank you, @rawkode!

/lgtm
/approve

**Pull request merged**